bin/proving-service/src/error.rs

```diff
@@ -18,6 +18,8 @@ pub enum ProvingServiceError {
     ParseError(#[from] std::num::ParseIntError),
     #[error("port {1} is already in use: {0}")]
     PortAlreadyInUse(#[source] std::io::Error, u16),
+    #[error("invalid prover type: {0}")]
+    InvalidProverType(String),
```

Comment:
Is this used? I was not able to find a reference but I might have missed something.

Reply:
nit: If it is used, the enum could be used as the inner field instead of a string

Reply:
It is not used, i just removed it.